packages/esm-patient-vitals-app/src/vitals/vitals-header/vitals-header.component.tsx

```diff
@@ -89,38 +89,23 @@ const VitalsHeader: React.FC<VitalsHeaderProps> = ({ patientUuid, showRecordVita
           <div className={styles['button-container']}>
             <Button className={styles['record-vitals']} kind="ghost" size="sm" onClick={launchVitalsAndBiometricsForm}>
               {t('recordVitals', 'Record vitals')}
-            </Button>
-            {showDetailsPanel ? (
-              <ChevronUp
-                size={16}
-                className={styles['collapse-button']}
-                title={'ChevronUp'}
-                onClick={(e) => {
-                  e.stopPropagation();
-                  toggleDetailsPanel();
-                }}
-              />
-            ) : (
-              <ChevronDown
+              <ArrowRight
                 size={16}
-                className={styles['expand-button']}
-                title={'ChevronDown'}
-                onClick={(e) => {
-                  e.stopPropagation();
-                  toggleDetailsPanel();
-                }}
+                className={styles['arrow-up-button']}
+                style={{ fill: '#0f62fe' }}
```

Comment:
Why use an inline style when you have a class, you could add the fill style to the `arrow-up-button` class in the .scss file.

Reply:
Also the color variable for that code is `$color-blue-60-2`